salt/_modules/metalk8s_kubernetes.py

```diff
@@ -351,3 +354,37 @@ def list_objects(kind, apiVersion, namespace='default', all_namespaces=False,
         raise CommandExecutionError('{}: {!s}'.format(base_msg, exc))
 
     return [obj.to_dict() for obj in result.items]
+
+
+def get_object_digest(object_key=None, checksum='sha256', *args, **kwargs):
+    """
+    Helper to get the digest of one kubernetes object or from a specific key
+    of this object (usefull to get the digest of one config from ConfigMap)
```

Comment:
One single object may not be sufficient in all cases: some Pods will for instance consume configuration data from both a Secret and a ConfigMap. But acceptable to keep as a debt ticket, as we don't need it for Dex.

Reply:
The call of this function is to retrieve one if you need to retrieve several then you should just do several calls on this one (and maybe create another module function to do it if you need to).

To me it's not a debt and not an issue it's just a feature we do not need right now so ....

Reply:
We're kind of forced to have an extensible method if we need more than one config to hash, as the hash should be stored in a single annotation. OK for the debt topic, let's wait for a need to implement this feature, you're right :)